statsmodels/base/model.py

```diff
@@ -1311,6 +1313,129 @@ def f_test(self, r_matrix, cov_p=None, scale=1.0, invcov=None):
                              invcov=invcov, use_f=True)
         return res
 
+    @staticmethod
+    def _is_lr_test_invalid(model, attribute, expected_value):
+        """
+        Likelihood test is not valid when:
+            1) cov_type of model is set to nonrobust
+            2) .fit(method=REML) in case of MixedLM
+
+        Parameters
+        ----------
+        model: Result instance
+
+        attribute: string
+            Attribute to be looked up in the input model.
+
+        expected_value: string
```

Comment:
expected expected value reminds me too much of of statistical expectation
maybe `valid_value` instead